strangerstudios/paid-memberships-pro
# PR1 - Adds checkboxes to the delete user page

### All Submissions:






### Changes proposed in this Pull Request:

Adds options to the delete user page, allowing you to choose if you want to remove active subscriptions, and delete member history for that user. 

### How to test the changes in this Pull Request:

1. Make sure a user has an active subscription
2. Navigate to Users and click delete on that user
3. You'll be given the option to cancel their active subscription and membership history

### Other information:




### Changelog entry

You can now choose if you want to remove an active subscription and delete membership history when deleting a user.
